kerastuner/engine/hyperparameters.py

```diff
@@ -377,9 +377,14 @@ class HyperParameters(object):
     """
 
     def __init__(self):
-        self.space = []
+        self._space = {}
```

Comment:
Add a comment to document the `_space` dict

Reply:
Done